src/app/core/public/api/impl/Circuit.ts

```diff
@@ -195,14 +195,27 @@ export abstract class CircuitImpl implements Circuit {
     // Wire connection can fail if i.e. p1 is reference-equal to p2
     public abstract connectWire(p1: Port, p2: Port): Wire | undefined;
 
+    /** Will delete all components and wires
+     * contained within the Obj[] array
```

Comment:
Let's also mention that it will disconnect any wires from components that are being deleted